.circleci/config.yml

```diff
@@ -151,7 +151,7 @@ jobs:
       # build only current commit (since we use squashed merging, this is safe)
       - run:
           name: Building, testing and uploading
-          command: bioconda-utils build recipes config.yml --anaconda-upload --git-range $CIRCLE_SHA1~1 $CIRCLE_SHA1
+          command: bioconda-utils build recipes config.yml --anaconda-upload --git-range $CIRCLE_SHA1~1 $CIRCLE_SHA1 --prelint
```

Comment:
Since we want to merge into `bulk`, I guess you also want to add this below to `bulk-linux`/`bulk-macos`, right?

Reply:
Thanks, good catch